mobile/build.gradle

```diff
@@ -142,7 +142,7 @@ dependencies {
     implementation "androidx.constraintlayout:constraintlayout:1.1.3"
     implementation "androidx.preference:preference:1.1.0"
     implementation "androidx.biometric:biometric:1.0.1"
-    implementation "com.google.android.material:material:1.1.0"
+    implementation "com.google.android.material:material:1.2.0-alpha04"
```

Comment:
This PR uses `material:1.2.0-alpha04`, so maybe I should wait with merging this.